### ISSUE TITLE
Speed up video processing

### ISSUE THREAD

**Issue by hepcat72** [hepcat72/flyMultiPlate#13]
The video driver appears to analyze the video at what it thinks is real time (as determined by the faulty fps and number of frames).  See if we can ignore the fps and process the video as fast as possible.

**Comment by hepcat72** [comment]
I tried not displaying the video, but the timing appears to be 60 seconds of video processed every 11 seconds.  It doesn't feel any faster.

I also need to figure out why the console message gets spit out twice in a row for 0 seconds processed.

**Comment by hepcat72** [comment]
The processing speed is not affected by the fps in the video file's metadata.  It's just going as fast as it can.  The only way to increase performance would be to rewrite the code.  I installed an editor that allowed me to change the fps from 30 to 60 and it still processes the frames at the same rate.

I don't think that showing the video really takes much extra time, but I'll leave the option to not show the live feed during reprocessing.

I also figured out why time 0 was output twice.  When the first frame is analyzed, it's compared with itself - same time stamp and everything, so the elapsed time is 0.  The next time through the loop, the "last time elapsed" is still 0 as a result and it prints again.  I changed the initial value of the last elapsed time to -1 and cleaned that up.
